app/assets/javascripts/admin/addon/components/schema-theme-setting/types/enum.gjs

```diff
@@ -5,7 +5,9 @@ import FieldInputDescription from "admin/components/schema-theme-setting/field-i
 import ComboBox from "select-kit/components/combo-box";
 
 export default class SchemaThemeSettingTypeEnum extends Component {
-  @tracked value = this.args.value || this.args.spec.default;
+  @tracked
+  value =
+    this.args.value || (this.args.spec.required && this.args.spec.default);
```

Comment:
Just confirming, will `this.args.spec` ever be nil?

Reply:
yup it'll never be null